test/functional/verify/ghosted/test.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+setup() {
+  clean_test_dir
+  create_manifest_tar 10 MoM
+  sign_manifest_mom 10
+  create_manifest_tar 10 os-core
+  chown_root -R "$DIR/target-dir"
+}
+
+teardown() {
+  clean_tars 10
+  clean_tars 10 files
+  sudo cp -r "$DIR/target-dir.bak/foo" "$DIR/target-dir"
+}
+
+@test "verify ghosted file skipped" {
+  run sudo sh -c "$SWUPD verify --fix $SWUPD_OPTS"
+
+  check_lines "$output"
+  [ $status -eq 0 ]
+  # this should exist at the end, despite being marked as "ghosted" in the
+  # Manifest. This is an important test because internally swupd treats ghosted
+  # files as deleted for rename purposes, but we need to make sure the file
```

Comment:
no renames are involved in this test

Reply:
I was just trying to explain the reason for the file existence test, but I'll clarify.